corehq/apps/export/tests/test_export_instance.py

```diff
@@ -356,7 +356,7 @@ def test_export_instance_from_saved(self):
             )
 
         self.assertEqual(len(instance.tables), 2)
-        self.assertEqual(len(instance.tables[0].columns), 2 + len(MAIN_FORM_TABLE_PROPERTIES))
+        self.assertEqual(len(instance.tables[0].columns), 3 + len(MAIN_FORM_TABLE_PROPERTIES))
```

Comment:
these got bumped because we accidentally would combine `@case_id` into one prop cause we didn't use transform for uniqueness: https://github.com/dimagi/commcare-hq/pull/11014/files#diff-9087195bf99fb8ba2fda775b0556aceaR707.

this number actually makes sense because there should be 3 more columns after sys props because that's what we define in the setup